docs/resources/dms_kafka_consumer_group.md

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_kafka_consumer_group
+
+Manages DMS Kafka consumer group resources within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+resource "huaweicloud_dms_kafka_consumer_group" "test" {
+  instance_id = var.instance_id
+  name        = "consumer_group_test"
+  description = "the description of the consumer group"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the kafka instance.
+
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the consumer group.
+
+  Changing this parameter will create a new resource.
+
+* `description` - (Optional, String) Specifies the description of the consumer group.
+
+* `state` - (Optional, String) Specifies the state of the consumer group.
+  Value options: **DEAD**, **EMPTY**, **PreparingRebalance**, **CompletingRebalance**, **Stable**.
```

Comment:
```suggestion
* `state` - Indicates the state of the consumer group.
  Value options: **DEAD**, **EMPTY**, **PreparingRebalance**, **CompletingRebalance**, **Stable**.
```
and fix other attributes of coordinator_id, lag and created_at

Reply:
done